src/api/app/components/sourcediff_component.html.haml

```diff
@@ -0,0 +1,37 @@
+.row
+  .col
+    - if @refresh
+      .clearfix.mb-2.text-center
+        .btn.btn-outline-primary.cache-refresh{ title: 'Refresh results', onclick: 'loadChanges();' }
```

Comment:
The `onclick` in this line is causing the issue I mentioned in my previous review.

Reply:
Yes. I changed `loadChanges` to not receive any parameter. This should fix it.